test/script/check-tag-alignment

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+exit_code=0
+
+# Documentation tags need to be aligned to the right margin, so look for
+# tags which aren't at the right margin.
+grep ' \*[^*]\+\*$' doc/ -r \
```

Comment:
I think it's debatable whether this is working as intended. Because of the leading space, tags on the far left margin are missed. If I remove it, I get these back:

```
doc/ale-development.txt:*ale-dev*
doc/ale-development.txt:*ale-development*
doc/ale-powershell.txt:*g:ale_powershell_psscriptanalyzer_executable*
doc/ale-powershell.txt:*g:ale_powershell_psscriptanalyzer_module*
doc/ale-powershell.txt:*g:ale_powershell_psscriptanalyzer_exclusions*
doc/ale-supported-languages-and-tools.txt:*ale-supported-list*
doc/ale.txt:*ale*
```

Some of those (ale-dev, ale) are heading tags and seem like they should potentially be allowed. But the [powershell](https://github.com/dense-analysis/ale/blob/master/doc/ale-powershell.txt#L30) examples seem pretty awkward and maybe not in the spirit of the check?